sdk/src/test/java/com/mapbox/maps/MapSurfaceTest.kt

```diff
@@ -0,0 +1,208 @@
+package com.mapbox.maps
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.view.MotionEvent
+import android.view.Surface
+import com.mapbox.maps.plugin.MapPlugin
+import com.mapbox.maps.renderer.MapboxSurfaceRenderer
+import com.mapbox.maps.renderer.OnFpsChangedListener
+import com.mapbox.maps.renderer.RendererSetupErrorListener
+import com.mapbox.maps.renderer.widget.BitmapWidget
+import com.mapbox.verifyOnce
+import io.mockk.*
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertNull
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
```

Comment:
just a question? is `RobolectricTestRunner` really needed here?

Reply:
These tests indeed pass with AndroidJUnit4